dolphinscheduler-alert/src/main/resources/mail_templates/alert_mail_template.ftl

```diff
@@ -14,4 +14,40 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
 -->
-<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01 Transitional//EN' 'http://www.w3.org/TR/html4/loose.dtd'><html><head><title> dolphinscheduler</title><meta name='Keywords' content=''><meta name='Description' content=''><style type="text/css">table {            margin-top:0px;            padding-top:0px;            border:1px solid;            font-size: 14px;            color: #333333;            border-width: 1px;            border-color: #666666;            border-collapse: collapse;        }        table th {            border-width: 1px;            padding: 8px;            border-style: solid;            border-color: #666666;            background-color: #dedede;        }        table td {            border-width: 1px;            padding: 8px;            border-style: solid;            border-color: #666666;            background-color: #ffffff;        }</style></head><body style="margin:0;padding:0"><table border="1px" cellpadding="5px" cellspacing="-10px"><thead><#if title??> ${title}</#if></thead><#if content??> ${content}</#if></table></body></html>
\ No newline at end of file
+<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01 Transitional//EN' 'http://www.w3.org/TR/html4/loose.dtd'>
```

Comment:
After formatting, you can verify whether the content of the sent email has spaces and blank lines.

Reply:
thx, i found this problem. i will fix

Reply:
I cancel format file: alert.resources.mail_templates.alert_mail_template.ftl

Reply:
Ok.